lib/v2/screens/send_scanner/send_scanner_screen.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/providers/services/navigation_service.dart';
+import 'package:seeds/screens/app/wallet/custom_transaction.dart';
+import 'package:seeds/v2/components/scanner/scanner_screen.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/screens/send_scanner/interactor/send_scanner_bloc.dart';
+import 'package:seeds/v2/screens/send_scanner/interactor/viewmodels/scanner_events.dart';
+import 'package:seeds/v2/screens/send_scanner/interactor/viewmodels/send_scanner_state.dart';
+
+/// Scanner SCREEN
```

Comment:
I also fail in this copy paste comment effect

Reply:
:)